setup.py

```diff
@@ -10,7 +10,7 @@
 except ImportError:
     pass
 
-VERSION = '0.4.6.2'
+VERSION = '0.4.6.3'
```

Comment:
Should we mark such increments with something work4-specific, to avoid potential conflicts with upstream/origin)?